source/opt/inline_pass.h

```diff
@@ -181,6 +182,12 @@ class InlinePass : public Pass {
   // Def/Use database
   std::unique_ptr<analysis::DefUseManager> def_use_mgr_;
 
+  // Update the DefUseManager when cloning decorations.
+  std::function<void(ir::Instruction&, bool)> UpdateDefUseMgr;
```

Comment:
It is a class variable, so I think correct naming should be update_def_use_mgr_.
Maybe callback_def_use_mgr_ for more clarity.

Reply:
Ah, it's not a callback.